en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/tracing-configurations.md

```diff
@@ -0,0 +1,282 @@
+# Choreo Connect Tracing Configuration Catalog
+
+This document describes all the configurations related to tracing that are used in WSO2 Choreo Connect.
+
+## Instructions for use
+
+Select the configuration sections, parameters, and values that are required for your use and add them to the `config.toml`
+file located in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/`.
+See the example .toml file given below.
+
+```toml
+# This is an example .toml file.
+[tracing]
+  enabled = true
+  type = "zipkin"
+
+[tracing.configProperties]
+    host = "zipkin"
+    port = "9411"
+    endpoint = "/api/v2/spans"
+    instrumentationName = "CHOREO-CONNECT"
+    maximumTracesPerSecond = "2"
+
+```
+
+
+
+
+
+
```

Comment:
Let's remove all this extra space.

Reply:
this is an auto generated file using config generator. It'll always generate with these spaces even if we remove it now.